app/views/alchemy/messages_mailer/contact_form_mail.text.erb

```diff
@@ -0,0 +1,3 @@
+<p>This is a placeholder template.</p>
+<p>Please create your own file: app/views/alchemy/messages_mailer/contact_form_mail.text.erb</p>
+<p>You can access all message fields using the @message instance variable.</p>
```

Comment:
The mailer is meant as an example. and uses the attributes that are present in the `alchemy/config.yml`. If someone alters them, then the attributes are not present anymore, but the mailer works out of the box. What about adding a note into the `config.yml` or updating the guides instead of deleting the files?

Reply:
I see what you mean.

I disagree, though, because this template is in the gem and never generated or touched by gem users. Since nothing mentions generating mail templates to match the `config`, I thought this would be a convenient way for developers (who don't automatically figure out the need to override your templates) to realize this when they test sending an email and see this language as the email body.

I'm happy to revert if you prefer.

Reply:
> because this template is in the gem and never generated or touched by gem users

This template is used until the user overwrites it in their Rails app.

We have a generator that generates the view into the Rails app.

We could add a comment into that Template so it makes it clear what needs to be adjusted if the mailer config changed the fields.

Reply:
This sounds good to me.